beacon-chain/state/stategen/replay.go

```diff
@@ -53,6 +53,10 @@ func (s *State) ReplayBlocks(ctx context.Context, state *stateTrie.BeaconState,
 // LoadBlocks loads the blocks between start slot and end slot by recursively fetching from end block root.
 // The Blocks are returned in slot-descending order.
 func (s *State) LoadBlocks(ctx context.Context, startSlot, endSlot uint64, endBlockRoot [32]byte) ([]*ethpb.SignedBeaconBlock, error) {
+	// Nothing to load for invalid range.
+	if endSlot < startSlot {
+		return nil, nil
```

Comment:
Return error?

Reply:
Discussed offline that this breaks many tests. We'll revisit at a later time. @nisdas suggested filing an issue and TODO for this, to fix the test input data.